app/posts/find-teacher-training/2021-08-18-visa-sponsorship-filter.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Visa sponsorship filter
+description: Why we added a new filter to Find
+date: 2021-08-18
+---
+
+Previous analysis showed us that a common reason for rejection is international candidates without the right to work or study in the UK applying for courses that are unable to sponsor visas.
+
+To address this, we’ve [asked providers whether they can sponsor visas](/publish-teacher-training-courses/visa-sponsorship/) and have included this within a [new ‘International students’ section](/find-teacher-training/new-international-students-section/).
+
+However, to save candidates from having to browse courses individually to find ones that will sponsor visas, we’ve also added a checkbox to the filters.
```

Comment:
We can simplify this by saying:

'We've also added a checkbox to the filters to help candidates find the courses that sponsor visas.'